.github/workflows/paportal-rolling-instance-actions.yml

```diff
@@ -82,6 +82,10 @@ jobs:
         website-id: f46b70cc-580b-4f1a-87c3-41deb48eb90d
         download-path: ./paportal-download
 
+    - name: Delete portal manifest before uploading
+      working-directory: ./paportal-download
+      run: rm ./starter-portal/.portalconfig/*-manifest.yml
```

Comment:
better to define a env variable (GH Flows' concept of 'pipeline' variables) once with the common path to the local starter portal folder, and the have the 2 delete steps and the upload/download steps use the common path from the env var expression